docs/content/en/1.getting-started/3.build.md

```diff
@@ -0,0 +1,17 @@
+# Build
+
+From v0.2.0, we shipped experimental build support with Vite. It's disabled by default and you can enable it by setting `vite.build: true` in config.
+
+```js
+// nuxt.config
+export default {
+  buildModules: [
+    'nuxt-vite'
+  ],
+  vite: {
+    build: true
```

Comment:
@pi0 Should we make it default to true and for users to opt-out? Or maybe in next breaking?

Reply:
I vote for 3.x so that we can fix leftover issues like modern mode mapping, speeding up and better legacy.